Example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "react": "~15.3.0",
-    "react-native": "^0.34.0",
+    "react-native": "^0.41.2",
```

Comment:
It's going to take me a while as I will want to figure out how to test my change with 0.34.0.  Will keep you posted.

p.s. I only have a nexus android.  Since the main change is for android, do I need to test for iOS before sending the pull request?